playbooks/technology_radar/data.csv

```diff
@@ -46,3 +46,6 @@ APIs as a product,assess,techniques,TRUE,
 GraphQL batching,assess,techniques,TRUE,
 Bug bounties,hold,techniques,FALSE,
 Serverless architecture,hold,techniques,TRUE,
+Relay,adopt,languages & frameworks,FALSE,
+Redux,assess,languages & frameworks,FALSE,
```

Comment:
@damassi While we’re using this in production (eg Prediction), with the newer React hooks API I _think_ the understanding is that in the future we still need to assess if we ever need to add full Redux again or if the builtin API is enough (like has been done in the new nav bar iirc). Is this correct?

Reply:
@alloy - I'm of the opinion that for anything app-like that contains significant amounts of state we should continue using Redux (Prediction, Position). It's patterns scale very well, the ecosystem is vast, and so much comes out of the box for free (such as performance tuning, middleware pipelines, devtools, etc). There's some really nice hooks-centric, typescript-friendly Redux libs out there, such as [easy-peasy](https://github.com/ctrlplusb/easy-peasy). 

For Most Artsy webpage-like code (think, Force "app shell", new nav bar, etc) hooks + context API is more than enough.

Reply:
That does seem to kinda follow my outline. I chose ‘asses’ to mean “do not use it by default, but if we determine that an app definitely benefits from a fuller redux-like env, only then do we use it”. But perhaps that should be ‘adopt’ and it’s implicitly clear that even for tech in ‘adopt’ we need to evaluate well on a case by case basis? (cc @joeyAghion)

Reply:
I think 
> Redux,assess,languages & frameworks,FALSE,

Is accurate 👍

Reply:
Done 👍